cypress/support/fragments/data_import/logs.js

```diff
@@ -0,0 +1,44 @@
+import { Accordion, Button, MultiColumnListCell, Selection, TextField } from '../../../../interactors';
+
+const collectionOfColumns = [MultiColumnListCell({ row: 0, columnIndex: 2 }), MultiColumnListCell({ row: 0, columnIndex: 3 }),
+  MultiColumnListCell({ row: 0, columnIndex: 4 }), MultiColumnListCell({ row: 0, columnIndex: 5 })];
+
+export default {
+  checkImportFile:(fileName) => {
+    cy.do(Button('View all').click());
+    cy.expect(TextField({ id:'input-job-logs-search' }).exists());
+    cy.do(TextField({ id: 'input-job-logs-search' }).fillIn('oneMarcBib.mrc'));
+    // TODO delete after fix by developers
+    cy.wait(20000);
+    cy.do(Button('Search').click());
+    cy.wait(20000);
+    cy.expect(Accordion({ id: 'jobProfileInfo' }).find(Button({ id: 'accordion-toggle-button-jobProfileInfo' })).exists());
+    cy.do(Accordion({ id: 'jobProfileInfo' }).find(Button({ id: 'accordion-toggle-button-jobProfileInfo' })).click());
+    cy.expect(Accordion({ id: 'jobProfileInfo' }).find(Selection()).exists());
+    cy.do(Accordion({ id: 'jobProfileInfo' }).find(Selection()).choose(fileName));
+  },
+
+  waitUntilSearchJobProfile:(jobProfileName) => {
+    cy.expect(MultiColumnListCell(jobProfileName).exists());
+  },
+
+  checkStatusOfJobProfile:() => {
+    cy.do(
+      MultiColumnListCell({ row: 0, columnIndex: 1 }).perform(element => {
+        expect(element).to.have.text('Completed');
```

Comment:
`MultiColumnListCell({ row: 0, column: 'Completed' }).exists()`

JFYI: another way for cheking coloumn text. This code do the same action but looks more shorter and understandable

Reply:
Thanks! I used your advice.